<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowSwitchZadacha</class>
 <widget class="QDialog" name="WindowSwitchZadacha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(170, 170, 255)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>568</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Yu Gothic UI&quot;;</string>
   </property>
   <property name="text">
    <string>Привіт, ми підскажемо тобі за скільки часу ти потрапиш на потрібну тобі станцію!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Yu Gothic UI&quot;;</string>
   </property>
   <property name="text">
    <string>Вирахувати</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>186</x>
     <y>79</y>
     <width>271</width>
     <height>146</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Yu Gothic UI&quot;;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QRadioButton" name="radioButton_3">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Yu Gothic UI&quot;;</string>
      </property>
      <property name="text">
       <string>Станцiя метро Casa de l’Aigua</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="radioButton">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Yu Gothic UI&quot;;</string>
      </property>
      <property name="text">
       <string>Станцiя метро Trinitat Nova</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QRadioButton" name="radioButton_5">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Yu Gothic UI&quot;;</string>
      </property>
      <property name="text">
       <string>Станцiя метро Can Cuias</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QRadioButton" name="radioButton_4">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Yu Gothic UI&quot;;</string>
      </property>
      <property name="text">
       <string>Станцiя метро Ciutat Meridiana</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="radioButton_2">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Yu Gothic UI&quot;;</string>
      </property>
      <property name="text">
       <string>Станцiя метро Torre Baro Vallbona</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
